src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/NodeBase.java

```diff
@@ -0,0 +1,152 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import edu.wpi.first.math.util.Units;
+
+import java.util.LinkedList;
+
+import static edu.greenblitz.tobyDetermined.RobotMap.TelescopicArm.Elbow.STARTING_ANGLE_RELATIVE_TO_GROUND;
+
+public class NodeBase  {
```

Comment:
almost everything in this class should be static 
the constructor can be replaced with a static block

Reply:
put things static

Reply:
gives null when try to get to a  value of a node